pkg/skaffold/initializer/analyze.go

```diff
@@ -105,14 +125,14 @@ type directoryAnalyzer struct {
 	currentDir string
 }
 
-func (a *directoryAnalyzer) analyzeFile(filePath string) error {
+func (a *directoryAnalyzer) analyzeFile(_ string) error {
```

Comment:
`_` can be removed, right?

Reply:
it's not going to implement Analyzer then :/

Reply:
I mean keep the argument but don’t give it a _ name.